external/mmat.m

```diff
@@ -51,9 +52,48 @@
 nDB = ndims(B);
 nD = max(nDA,nDB);
 
+if nD == 2 && isdefaultdim
+    C = A * B;
+    return
+end
+
 nA = [size(A),ones(1,nD-nDA)]; nA = nA(dim); 
 nB = [size(B),ones(1,nD-nDB)]; nB = nB(dim);
 
+% Array is double float which is 8 bytes per element
+neededMem = (numel(A)*nB(2) + numel(B)*nA(1)) * 8;
```

Comment:
See comment as this might need to be multiplied by 2

Reply:
Ah that's what it was before but we didn't know why! Worth a comment in the code as well I think for future us!